packages/canon/css/components.css

```diff
@@ -94,6 +94,36 @@
   height: 1.5rem;
 }
 
+.canon-DataTableRoot {
```

Comment:
why are these styles not in the `datatable.css` file?

Reply:
They are. Unfortunately for now every CSS has to be pushed in 3 places. We make CSS close to each components and then we run `yarn build:css` to build the `component.css` and `styles.css` files.